roles/matrix-synapse/vars/workers.yml

```diff
@@ -206,7 +206,7 @@ matrix_synapse_workers_generic_worker_endpoints:
   # You might also wish to investigate the `update_user_directory` and
   # `media_instance_running_background_jobs` settings.
 
-# pusher worker (no API endpoints) [
+  # pusher worker (no API endpoints) [
```

Comment:
This file is automatically generated (see `roles/matrix-synapse/files/workers-doc-to-yaml.sh`).

The next time we run that file to regenerate `workers.yml`, these changes will be thrown out.